code/modules/mapping/mapping_helpers.dm

```diff
@@ -205,3 +205,24 @@ INITIALIZE_IMMEDIATE(/obj/effect/mapping_helpers/no_lava)
 		CRASH("Wrong disease type passed in.")
 	var/datum/disease/D = new disease_type()
 	return list(component_type,D)
+
+/obj/effect/mapping_helpers/dead_body_placer
+	name = "Dead Body placer"
+	icon_state = "deadbodyplacer"
+	var/bodycount = 2 //number of bodies to spawn
+
+/obj/effect/mapping_helpers/dead_body_placer/Initialize(mapload)
```

Comment:
Should probably `LateInitialize()` so every trays in the area are instanciated before trying to put corpses in them.

Reply:
Thanks, that did it.